src/extension.ts

```diff
@@ -605,17 +605,33 @@ function continueEditing(outline: DocumentOutline, useContext: boolean, selectio
 			if (lineRange.start.isAfterOrEqual(userStart)) {
 				break;
 			}
-			activeLineTexts.push(
-				outdentQuote(
-					document.getText(new vscode.Range(
-						lineRange.start,
-						lineRange.end.isAfterOrEqual(userStart)
-							? document.positionAt(Math.max(document.offsetAt(userStart) - 1, 0))
-							: lineRange.end
-					)),
-					lineRange.quoteIndent
-				)
+			const lineTexts = outdentQuote(
+				document.getText(new vscode.Range(
+					lineRange.start,
+					lineRange.end.isAfterOrEqual(userStart)
+						? document.positionAt(Math.max(document.offsetAt(userStart) - 1, 0))
+						: lineRange.end
+				)),
+				lineRange.quoteIndent
 			);
+
+			// Find lines with `@import "file.md"` and append its content to the active context.
+			for (const line of lineTexts.split(/\r?\n/)) {
+				const match = line.match(/\@import\s+\"([^\"]+)\"/);
+				if (match !== null && vscode.workspace.workspaceFolders !== undefined) {
+					const workspaceFolder = vscode.workspace.workspaceFolders[0].uri;
+					const filename = match[1];
+					const fullPath = vscode.Uri.joinPath(workspaceFolder, filename);
+					try {
+						const doc = await vscode.workspace.openTextDocument(fullPath);
+						activeLineTexts.push(doc.getText());
+					} catch {
+						vscode.window.showErrorMessage(`Error occurred while importing ${filename}`);
```

Comment:
Please rewrite ``` `Error occurred while importing ${filename}` ``` considering localization as `l10n.t('Error occurred while importing {0}', filename)`.
Additionally, try adding the relevant lines to `package.nls.json`, `package.nls.ja.json`, and `package.nls.cn.json`.
Using Markdown Copilot can simplify the translation into Simplified Chinese.

see: [@vscode/l10n](https://github.com/microsoft/vscode-l10n/tree/main/l10n)